modules/user-access/pages/access-control-model.adoc

```diff
@@ -85,17 +88,18 @@ GSQL offers 5 built-in local roles and 2 built-in global roles. The built-in rol
 
 === User-defined roles
 
-Users can define roles with their own list of privileges they want to grant to the role.To learn how to create/drop user-defined roles and manage privileges for the roles, see xref:role-management.adoc[Role Management].
+Users can define roles with their own list of privileges they want to grant to the role.
+To learn how to create/drop user-defined roles and manage privileges for the roles, see xref:role-management.adoc[Role Management].
 
 
 [#_access_control_lists]
 == Access control lists
-ACLs give you the ability to apply finer-grain level of access to GSQL queries.
-ACLs are bound to queries, and specify who can read or execute the query.
+ACLs give you the ability to apply finer-grained levels of access to a GSQL query by specifying who can read or execute the query.
+ACLs are bound to queries.
 
-An ACL contains two entries - `READ` and `EXECUTE` - for read privilege and execute privilege on a given query respectively.
+An ACL contains two entries - `READ` and `EXECUTE` - for read privilege and execute (run) privilege on a given query respectively.
```

Comment:
There should be a comma before "respectively", i.e., "for read privilege and execute (run) privilege on a given query, respectively."
See https://www.advancedenglishediting.com/blog/respectively-how-to-use-it-in-your-writing